src/sage/categories/lambda_bracket_algebras_with_basis.py

```diff
@@ -63,10 +63,11 @@ class FinitelyGeneratedAsLambdaBracketAlgebra(CategoryWithAxiom_over_base_ring):
 
         EXAMPLES::
 
-            sage: C = LieConformalAlgebras(QQbar)
-            sage: C.WithBasis().FinitelyGenerated()
-            Category of finitely generated Lie conformal algebras with basis over Algebraic Field
-            sage: C.WithBasis().FinitelyGenerated() is C.FinitelyGenerated().WithBasis()
+            sage: C = LieConformalAlgebras(QQbar)                                           # optional - sage.rings.number_field
+            sage: C.WithBasis().FinitelyGenerated()                                         # optional - sage.rings.number_field
+            Category of finitely generated Lie conformal algebras with basis
+             over Algebraic Field
+            sage: C.WithBasis().FinitelyGenerated() is C.FinitelyGenerated().WithBasis()    # optional - sage.rings.number_field
```

Comment:
Maybe something like this works:
```suggestion
            sage: C = LieConformalAlgebras(QQbar).WithBasis().FinitelyGenerated(); C   # optional - sage.rings.number_field
            Category of finitely generated Lie conformal algebras with basis
             over Algebraic Field
            sage: C is C.FinitelyGenerated().WithBasis()                               # optional - sage.rings.number_field
```